setup.py

```diff
@@ -51,9 +47,9 @@ def read(*rnames):
     author_email='zope-dev@zope.org',
     description="Zope table",
     long_description=(
-        read('README.txt')
+        open('README.txt').read()
         + '\n\n' +
-        read('CHANGES.txt')
+        open('CHANGES.txt').read()
```

Comment:
This will produce ResourceWarnings about unclosed files in some circumstances (e.g. PYTHONWARNINGS=all).  It's best to keep the `def read(...)` helper and make it use a `with open(...) as f: return f.read()`.

Reply:
done